doc/rst/language/spec/records.rst

```diff
@@ -409,36 +411,9 @@ Copy Initialization of Records
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When a new record variable is created based upon an existing variable,
-we say that the new variable is *copy initialized* from the existing
-variable. For example: 
-
-.. code-block:: chapel
-
-   var x: MyRecordType;
-   var y = x; // this is called copy initialization
-
-We say that y is *copy initialized* from x. Since x and y both exist
-after y is initialized and they are different record variables, they
-must not be observed to alias. That is, modifications to a field in x
-should not cause the corresponding field in y to change.
-
-Initializing a record variable with the result of a call returning a
-record by value simply captures the result of the call into the variable
-and does not cause copy initialization. For example: 
-
-.. code-block:: chapel
-
-   proc returnsRecord() {
-     var ret: MyRecordType;
-     return ret;
-   }
-   var z = returnsRecord(); // captures result into z without copy initializing
-
-..
-
-   *Future*.
-
-   Specifying further details of copy initialization is future work.
+it is copy initialized or move initialized as described in
```

Comment:
I have a slight preference for putting these two terms in something like quotes or italics.